nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/GetHDFSFileInfo.java

```diff
@@ -707,24 +642,24 @@ void finish(ProcessSession session) {
     /*
      * Keeps all request details in single object.
      */
-    static class HDFSFileInfoRequest{
-        enum Groupping {
+    static class HDFSFileInfoRequest {
+        protected enum Grouping {
```

Comment:
As the enclosing class is `package private`, I don't think `protected` really makes sense here. I believe the original was fine. Or do I miss something?